### PR TITLE
Email Notifications: Add StartTLSPolicy config flag

### LINKED ISSUE #24563: Still unable to send emails that do not require TLS connections
As per Ref https://github.com/grafana/grafana/issues/7189

It should be possible to send emails to an SMTP server that doesn't require TLS connections.

This is still an issue and experienced by 3 other users in the previously closed issue. 

Grafana Version v6.7.1

Config (helm chart):
```
  smtp:
    enabled: true
    host: "10.10.120.18:25"
    skip_verify: true
    from_address: "grafana@grafana.co.za"
    from_name: "Grafana"
    ehlo_identity: "server.grafana.local"
```
```
Log Error:
t=2020-03-24T14:02:29+0000 lvl=eror msg="failed to send notification" logger=alerting.notifier uid= error="Failed to send notification to email address: <EMAILADD>: 454 4.7.0 TLS not available due to local problem"
```

### PR DESCRIPTION
**Pull request by thameezb**
Fixes #24563

Ensures that current default behaviour is maintained if config arg is not set.

### CODE REVIEW

**Comment by thameezb** [comment]
Did not find any appropriate test locations to add any tests to

File: pkg/setting/setting_smtp.go [hunk 1]
```diff
@@ -27,6 +28,7 @@ func (cfg *Cfg) readSmtpSettings() {
 	cfg.Smtp.FromAddress = sec.Key("from_address").String()
 	cfg.Smtp.FromName = sec.Key("from_name").String()
 	cfg.Smtp.EhloIdentity = sec.Key("ehlo_identity").String()
+	cfg.Smtp.EhloIdentity = sec.Key("startTLS_policy").String()
```
Comment (bergquist) [inline pkg/setting/setting_smtp.go:31]: This needs to be added to `/conf/default.ini` and `conf/sample.ini` and `docs/source/installation/configuration.md`

**Review by bergquist — CHANGES_REQUESTED** [review]
Thank you for contributing to Grafana. This new setting should be documented.

File: pkg/setting/setting_smtp.go (hunk 1 above)
Comment (thameezb) [reply pkg/setting/setting_smtp.go]: Thanks. 
Added

**Comment by bergquist** [comment]
Thank you for contributing to Grafana! 

This will be released in Grafana 7.1

**Review by bergquist — APPROVED** [review]

**Pull request merged**